### PR TITLE
feat: Ability to sendDocument formed from bytes

### PR DESCRIPTION
**Pull request by akahan**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -41,5 +41,5 @@ pub mod objects;
 mod parse_mode;
 
 pub use api_params::*;
-pub use objects::*;
+// pub use objects::*;
```
Comment (ayrat555) [inline src/lib.rs:44]: can you please revert this change

File: src/api/telegram_api_impl.rs [hunk 2]
```diff
@@ -245,10 +257,12 @@ mod tests {
     use crate::api_params::StopPollParams;
     use crate::api_params::UnbanChatMemberParams;
     use crate::api_params::UnpinChatMessageParams;
+    use crate::objects::AllowedUpdate;
     use crate::objects::BotCommand;
     use crate::objects::ChatPermissions;
     use crate::objects::InlineQueryResultVenue;
-    use crate::AllowedUpdate;
+
+    use super::*;
```
Comment (ayrat555) [inline src/api/telegram_api_impl.rs:265]: can you please revert this change

**Review by ayrat555 — COMMENTED** [review]
looking ok to me

@EdJoPaTo @pxp9 can you please check also

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]
I like seeing the creation of a File via Buffer instead of a file on disk!

There is a bunch if clippy hints and @ayrat555 also suggested some things I would suggest too.

There is also a lot of seemingly unrelated change, mainly code formatting which should be its own PQ to simplify reviewing of the concise change of creating `File` from bytes.

File: src/lib.rs (hunk 1 above)
Comment (akahan) [reply src/lib.rs:44]: ok. but `#[warn(ambiguous_glob_reexports)]` on by default.

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]
There are still some linter hints which should be fixed. Also this PR will probably have merge conflicts with #109 as both are file related?

**Comment by ayrat555** [comment]
closed due inactivity

**Pull request closed without merging**